tokio-console/src/config.rs

```diff
@@ -238,3 +312,86 @@ impl FromStr for RetainFor {
         }
     }
 }
+
+// === impl ColorToggles ===
+
+impl ColorToggles {
+    pub fn color_durations(&self) -> bool {
+        self.color_durations.map(std::ops::Not::not).unwrap_or(true)
+    }
+
+    pub fn color_terminated(&self) -> bool {
+        self.color_durations.map(std::ops::Not::not).unwrap_or(true)
```

Comment:
minor style nit: `Not` is imported, so we could just write
```suggestion
        self.color_durations.map(Not::not).unwrap_or(true)
    }

    pub fn color_terminated(&self) -> bool {
        self.color_durations.map(Not::not).unwrap_or(true)
```

Reply:
it might also be worth having a comment or something here explaining why these are inverted?